src/test/java/kitchenpos/application/ProductServiceTest.java

```diff
@@ -0,0 +1,190 @@
+
+        when(purgomalumClient.containsProfanity(any())).thenReturn(true);
+
+        assertThrows(IllegalArgumentException.class, () -> productService.create(request));
+    }
+
+    @DisplayName("`상품`의 가격을 변경할 수 있다")
+    @Test
+    void changeProductPriceWithValidInput() {
+        Product request = new Product();
+        request.setName("상품 이름");
+        request.setPrice(BigDecimal.valueOf(2000));
+
+        Product existingProduct = new Product();
+        existingProduct.setId(UUID.randomUUID());
+        existingProduct.setName("상품 이름");
+        existingProduct.setPrice(BigDecimal.valueOf(3000));
+
+        MenuProduct menuProduct = new MenuProduct();
+        menuProduct.setProduct(existingProduct);
+        menuProduct.setQuantity(1);
+
+        Menu menu = new Menu();
+        menu.setMenuProducts(List.of(menuProduct));
+        menu.setPrice(BigDecimal.valueOf(1500));
+        menu.setDisplayed(true);
+
+        when(productRepository.findById(any())).thenReturn(java.util.Optional.of(existingProduct));
+        when(menuRepository.findAllByProductId(any())).thenReturn(List.of(menu));
+
+        productService.changePrice(existingProduct.getId(), request);
+
+        assertThat(existingProduct.getPrice()).isEqualTo(request.getPrice());
+        assertThat(menu.isDisplayed()).isTrue();
+    }
+
+    @Test
+    void changeProductPriceWithNullPrice() {
+        Product request = new Product();
+        request.setName("상품 이름");
+        request.setPrice(null);
+
+        assertThrows(IllegalArgumentException.class,
+            () -> productService.changePrice(UUID.randomUUID(), request));
+    }
+
+    @Test
+    void changeProductPriceWithNonExistingProduct() {
+        Product request = new Product();
+        request.setName("상품 이름");
+        request.setPrice(BigDecimal.valueOf(1000));
+
+        when(productRepository.findById(any())).thenReturn(java.util.Optional.empty());
+
+        assertThrows(NoSuchElementException.class,
+            () -> productService.changePrice(UUID.randomUUID(), request));
+    }
+
+    @DisplayName("'상품'의 가격이 변경되어, '상품'이 포함된 기존 '메뉴' 가격보다 높으면 숨김 처리 된다.")
+    @Test
+    void changeProductPriceWithHigherPriceThanMenu() {
+        Product request = new Product();
+        request.setName("상품 이름");
+        request.setPrice(BigDecimal.valueOf(1000));
+
+        Product existingProduct = new Product();
+        existingProduct.setId(UUID.randomUUID());
+        existingProduct.setName("상품 이름");
+        existingProduct.setPrice(BigDecimal.valueOf(2000));
+
+        MenuProduct menuProduct = new MenuProduct();
+        menuProduct.setProduct(existingProduct);
+        menuProduct.setQuantity(1);
+
+        Menu menu = new Menu();
+        menu.setMenuProducts(List.of(menuProduct));
+        menu.setPrice(BigDecimal.valueOf(1500));
+        menu.setDisplayed(true);
+
+        when(productRepository.findById(any())).thenReturn(java.util.Optional.of(existingProduct));
```

Comment:
<img width="436" alt="image" src="https://github.com/next-step/ddd-legacy/assets/47850258/28b83e13-6b31-4b06-bae7-f572b3e6a457">

import를 해주시면 어떨까요? 🙂

```suggestion
        when(productRepository.findById(any())).thenReturn(Optional.of(existingProduct));
```